src/h2/utilities.py

```diff
@@ -345,30 +323,26 @@ def _reject_pseudo_header_fields(headers, hdr_validation_flags):
     method = None
 
     for header in headers:
-        if _custom_startswith(header[0], b':', u':'):
+        if header[0][0] == SIGIL:
```

Comment:
If we want to go down the `.startswith()` path, we should align this line with that too. This also probably means the top-level `SIGIL` and `INFORMATIONAL_START` become obsolete.